.gitignore

```diff
@@ -1,12 +1,6 @@
 # If a change is made in here, add it to .npmignore as well unless you
 # explicitly want to include it in the published artifact
-node_modules/*
-!node_modules/@okta
-node_modules/@okta/*
-!node_modules/@okta/i18n
-!node_modules/@okta/courage
-!node_modules/@okta/jquery.simplemodal
-!node_modules/@okta/qtip2
+node_modules
```

Comment:
Yep, although this will be a **manual** process for now until we get Lerna or write a script - updating one of these dependencies means installing it like normal, and then copying the files over to packages (and in the case of i18n, limiting the copy to the properties files).